ledger/txtail.go

```diff
@@ -45,55 +85,56 @@ type txTail struct {
 	lowWaterMark basics.Round // the last round known to be committed to disk
```

Comment:
comment above should probably be "duplicate detection only works for queries with LastValid > lowWaterMark"

Reply:
no, b/c:
1. `lowWaterMark` is set to `rnd` at
```golang
	for ; t.lowWaterMark < rnd; t.lowWaterMark++ {
		delete(t.lastValid, t.lowWaterMark)
	}
```
2. duplicate detection is run for `lastValid >= lowWaterMark`
```golang
	if lastValid < t.lowWaterMark {
		return &errTxTailMissingRound{round: lastValid}
	}
```